src/installer/managed/Microsoft.NET.HostModel/Bundle/Bundler.cs

```diff
@@ -32,6 +32,7 @@ public class Bundler
         public readonly Manifest BundleManifest;
         private readonly TargetInfo Target;
         private readonly BundleOptions Options;
+        private readonly bool _macosCodesign;
```

Comment:
Nit: I know the style in the file is weird, but I would prefer for it to be consistent over any specific style (or we could change the style for everything in the file to be consistent)